app/build.gradle

```diff
@@ -4,12 +4,14 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
+apply plugin: 'kotlin-kapt'
+
 android {
     compileSdkVersion targetSdkVersion
     buildToolsVersion toolsVer
     defaultConfig {
         applicationId appId
-        minSdkVersion minSdkVersion
+        minSdkVersion minimalSdkVersion
```

Comment:
Please revert this change.

Reply:
Gradle takes value from DefaultProductFlavor.class so I changed name to minSdkVer